.github/workflows/sendJsonFile.yaml

```diff
@@ -0,0 +1,93 @@
+name: Send json file
+
+on:
+  issues:
+    types: 
+      - opened
+      - reopened
+      - edited
+
+jobs:
+  check-addon:
+    name: Check add-on
+    runs-on: windows-latest
+    steps:
+    - name: Get data
+      id: get-data
+      uses: actions/github-script@v6
+      with:
+        script: |
```

Comment:
what happens when an issue is opened that does not match this format?

Reply:
>Regarding issues not formatted according to this template, in this case the GitHub Actions workflow will fail. I don't know how to make it to run just for issues opened with this template. (https://github.com/nvaccess/addon-datastore/pull/37#issuecomment-1455521000)

Can you filter these issues by label? Can you make the label "autoSubmissionFromIssue"?